bigquery/google/cloud/bigquery/_helpers.py

```diff
@@ -209,6 +209,7 @@ def _row_tuple_from_json(row, schema):
 
 def _rows_from_json(values, schema):
     """Convert JSON row data to rows with appropriate types."""
+    # TODO(alixh) remove circular import
```

Comment:
Please don't use a `TODO`, instead use the issue tracker.

Reply:
Reverted this commit in cac00de and created a issue #4305